packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -196,10 +196,6 @@ class AndroidDevice extends Device {
     return '/data/local/tmp/sky.${app.id}.sha1';
   }
 
-  String _getDeviceApkSha1(ApplicationPackage app) {
-    return runCheckedSync(adbCommandForDevice(<String>['shell', 'cat', _getDeviceSha1Path(app)]));
-  }
-
   String _getSourceSha1(ApplicationPackage app) {
```

Comment:
Yep, to write the SHA1 to the device during install. And we'll want to use that again, but correctly this time - #8295.